cfpb/cfgov-refresh
# Add Regulations 3000 Skeleton

This PR adds the `regulations3k` package and a `REGULATIONS3K` feature flag that is on by default for `DEPLOY_ENVIRONMENT=build`. When `REGULATIONS3K` is enabled, Regulations 3000 will be served (for now) at `/regulations3k`.

This skeleton is based on the https://github.com/cfpb/cfgov-satellite skeleton app.

## Additions

- `REGULATIONS3K` feature flag
- `regulations3k` package

## Testing

1. Add `DEPLOY_ENVIRONMENT=build` to .python_env (for docker) or .env
2. Visit `/regulations3k` and see a "Hello, world!" page

## Todos

- Figure out if `regulations3k/css` and `regulations3k/js` are the correct places for these files within the cfgov-refresh context. (@contolini)

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
